lib/senders/templates/verify_primary.txt

```diff
@@ -0,0 +1,12 @@
+{{t "Verify primary email"}}
+{{t "A request to perform an account change has been made from the following device:"}}
```

Comment:
This template needs some vertical whitespace to separate these lines, it's quite a hard-to-read jumbled mass when you view the rendered output:

```
Verify primary email
A request to perform an account change has been made from the following device:
Firefox on Mac OSX 10.11
Madrid, Spain (estimated)
IP address: 10.246.67.38
Verify email:  http://127.0.0.1:3030/complete_signin?code=ae35999f861ffc81d594034eb4560af8&uid=6510cb04abd742c6b3e4abefc7e39c9f&service=sync&redirectTo=https%3A%2F%2Fredirect.com%2F&resume=eyJjYW1wYWlnbiI6bnVsbCwiZW50cnlwb2ludCI6bnVsbCwiZmxvd0lkIjoiM2Q1ODZiNzY4Mzc2NGJhOWFiNzhkMzMxMTdjZDU4Y2RmYjk3Mzk5MWU5NTk0NjgxODBlMDUyMmY2MThhNmEyMSIsInJlc2V0UGFzc3dvcmRDb25maXJtIjp0cnVlLCJ1bmlxdWVVc2VySWQiOiI1ODNkOGFlYS00NzU3LTRiZTQtYWJlNC0wZWQ2NWZhY2Y2YWQiLCJ1dG1DYW1wYWlnbiI6bnVsbCwidXRtQ29udGVudCI6bnVsbCwidXRtTWVkaXVtIjpudWxsLCJ1dG1Tb3VyY2UiOm51bGwsInV0bVRlcm0iOm51bGx9&utm_source=email&utm_medium=email&utm_campaign=fx-welcome-primary&utm_content=fx-activate

This is an automated email;if you did not authorize this action, then <a href="http://127.0.0.1:3030/settings/change_password?email=testuser%40testuser.com&utm_source=email&utm_medium=email&utm_campaign=fx-welcome-primary&utm_content=fx-change-password" style="color: #0095dd; text-decoration: none; font-family: sans-serif;">please change your password</a>.
For more information, please visit <a href="https://support.mozilla.org/kb/im-having-problems-with-my-firefox-account?utm_source=email&utm_medium=email&utm_campaign=fx-welcome-primary&utm_content=fx-support" style="color: #0095dd; text-decoration: none; font-family: sans-serif;">Mozilla Support</a>.s

Mozilla. 331 E Evelyn Ave, Mountain View, CA 94041
Mozilla Privacy Policy https://www.mozilla.org/privacy?utm_source=email&utm_medium=email&utm_campaign=fx-welcome-primary&utm_content=fx-privacy
```